pylint/checkers/classes/class_checker.py

```diff
@@ -258,6 +264,30 @@ def _has_different_parameters(
     return result
 
 
+def _has_different_keyword_parameters(
+    original: List[nodes.AssignName],
+    overridden: List[nodes.AssignName],
+) -> List[str]:
+    """Determine if the two methods have different keyword only parameters."""
+    original_names = [i.name for i in original]
+    overridden_names = [i.name for i in overridden]
+
+    if any(name not in overridden_names for name in original_names):
+        return ["Number of parameters "]
+
+    for name in overridden_names:
+        if name in original_names:
+            continue
+
+        try:
+            overridden[0].parent.default_value(name)
+            continue
```

Comment:
This is straight-up bikeshedding, but what's your view on removing the `continue` here and in the similar place above?

Reply:
I removed this one, the other leads to a crash. We need the `if not original_param:` check and without the `continue` it becomes meaningless.